docs/modules/operation/pages/events/sources/kafka.adoc

```diff
@@ -59,3 +59,4 @@ config:property-set eventsTopic opennms-kafka-events
 config:update
 ----
 
+NOTE: While configuring events topic, make sure that it doesn't conflict with other topics in OpenNMS subsystem. If you are unsure, keep the default one.
```

Comment:
NOTE: While configuring the eventsTopic, make sure that it doesn't conflict with other topics in the OpenNMS subsystem. If you are unsure, keep the default one.

Reply:
```suggestion
NOTE: While configuring the eventsTopic, make sure that it doesn't conflict with other topics in the OpenNMS subsystem. If you are unsure, keep the default one.
```